packages/camera/camera/ios/Classes/FLTCam.m

```diff
@@ -92,6 +97,23 @@ - (instancetype)initWithCameraName:(NSString *)cameraName
                        orientation:(UIDeviceOrientation)orientation
                captureSessionQueue:(dispatch_queue_t)captureSessionQueue
                              error:(NSError **)error {
+  AVCaptureSession *captureSession = [[AVCaptureSession alloc] init];
```

Comment:
Optional nit: There's no reason to make a local variable for this; you can just inline the alloc/init in the method call.